README.md

```diff
@@ -311,6 +311,139 @@ export default function() {
 }
 ```
 
+#### Locator API
+
+We suggest using the Locator API instead of the low-level
+ElementHandle methods. An element handle can go stale if
+the element's underlying frame is changed. However,
+with the Locator API, even if the underlying frame
+changes, locators will work.
+
+Here's an example to better understand the differences.
+
+```js
+import launcher from "k6/x/browser";
+
+export default function () {
+  const browser = launcher.launch('chromium', {
+    headless: __ENV.XK6_HEADLESS ? true : false,
+  });
+  const context = browser.newContext();
+  const page = context.newPage();
+  page.goto("https://test.k6.io/flip_coin.php", {
+    waitUntil: "networkidle",
+  });
+
+  /*
+  In this example, we will use two locators, matching a
+  different betting button (head and tail) on the page.
```

Comment:
Nit: the values are actually "heads" and "tails", i.e. it's never used as singular.

You might want to change the variable names as well.